### PR TITLE
Podcast: add 238

### PR DESCRIPTION
**Pull request by bitschmidty**

### CODE REVIEW

File: _posts/en/podcast/2023-02-16-newsletter-recap.md [hunk 1]
```diff
@@ -0,0 +1,1242 @@
...
+
+**Mike Schmidt**: Seardsalmon?
+
+**Vivek Kasarabada**: Hi, my name's Vivek.  I do corporate development for
+Blockstream, so not a dev, and that's why I need Alex!
+
+**Mike Schmidt**: Well, thanks for joining us.  Alex, do you want to introduce
+yourself?
+
+**Alex** **Myers**: Sure, yeah, I'm at Blockstream working on Core Lightning.
```
Comment (Xekyo) [inline _posts/en/podcast/2023-02-16-newsletter-recap.md:90]:
```suggestion
**Alex Myers**: Sure, yeah, I'm at Blockstream working on Core Lightning.
```

File: _posts/en/podcast/2023-02-16-newsletter-recap.md [hunk 2]
```diff
@@ -0,0 +1,1242 @@
...
+a massive demand for this, and the perception I get from talking to people who
+like this stuff is that the whole point of doing this the way that they're doing
+and talking to Casey as well is that they want to do this the Bitcoin way, which
+is very anchored into the chain, and the inscriptions are done once, and they're
+put in a block and they don't get moved, the sats' ownership moves, and that's
+how they perceive the scarcity value of this, is the fact that it is a very
+early block in their ordinal system thing.  So, that's why they're not
+interested in doing it offchain.
+
+That was essentially why so many of the people doing this craft on Ethereum,
```
Comment (Xekyo) [inline _posts/en/podcast/2023-02-16-newsletter-recap.md:383]: I suspect that “craft” is not the word that Rodolfo said here, but probably “crap” or “cruft”?

**Review by Xekyo — COMMENTED** [review]
Looked over the anchors and links, looks reasonable. Briefly scrolled over the transcript, noticed just a couple things. Also, I was wondering whether we should mention people that spontaneously join the conversation such as Rodolfo in this case.

**Review by Xekyo — APPROVED** [review]

File: _posts/en/podcast/2023-02-16-newsletter-recap.md (hunk 2 above)
Comment (bitschmidty) [reply _posts/en/podcast/2023-02-16-newsletter-recap.md:383]: crap it is

File: _posts/en/podcast/2023-02-16-newsletter-recap.md (hunk 1 above)
Comment (bitschmidty) [reply _posts/en/podcast/2023-02-16-newsletter-recap.md:90]: Good catch

**Comment by bitschmidty** [comment]
> I was wondering whether we should mention people that spontaneously join the conversation such as Rodolfo in this case.

I think case by case we can consider it. Im going to omit him from the summary in this episode.

Pushed changes for your comments

**Comment by Xekyo** [comment]
Good to go, I think

**Pull request merged**